### DISCUSSION TITLE
Add a PAC AUTH Refresh Option for Refreshing MFA

### DISCUSSION THREAD

**Discussion by daryllabar** [microsoft/powerplatform-build-tools#1059] [Ideas]
PAC AUTH should have a Refresh option to refresh credentials, (I'm thinking of MFA, but there could be other login information that needs to be refreshed)

I have a script I run to deploy my plugins to my dev environment.  It utilizes a named connection and failed today with this error:

```
Error: Failed to connect to Dataverse
    ExternalTokenManagement Authentication Requested but not configured correctly. 003
        AADSTS50078: Presented multi-factor authentication has expired due to policies configured by your administrator, you must refresh your multi-factor authentication to access '00000007-0000-0000-c000-000000000000'. Trace ID: 712a623a-db26-4819-9671-fa9d7e080000 Correlation ID: 6a9ce1e5-5d76-4193-b7d1-7afb8214ff80 Timestamp: 2024-10-15 21:04:28Z
            AADSTS50078: Presented multi-factor authentication has expired due to policies configured by your administrator, you must refresh your multi-factor authentication to access '00000007-0000-0000-c000-000000000000'. Trace ID: 712a623a-db26-4819-9671-fa9d7e080000 Correlation ID: 6a9ce1e5-5d76-4193-b7d1-7afb8214ff80 Timestamp: 2024-10-15 21:04:28Z
```

It wasn't clear on how to refresh the MFA.  I was able to create a new connection with the same account, and it solved my MFA issue with the connection that previously existed, but it would make sense to allow same way to refresh the MFA, rather than having to temporarily create, and then delete, another connection with the same credentials.

*Note*  maybe similar to #706

**Comment by rajyraman** [comment]
I have the same issue from time to time, which I work around by running _pac auth create_ and then deleting the new profile.

**Reply by daryllabar** [reply]
This was my work around as well.  It wasn't intuitive, and if someone accidently creates a new one, and doesn't notice that they are now pointing to a different Org, they could potentially deploy something to the wrong environment which could be all sorts of bad.

**Reply by rajyraman** [reply]
@daryllabar - I always use --environment switch because I am really scared of screwing up something. It is a pain to add this to every command but it saves me from doing pac auth list and figure out what is the active profile.
